### PR TITLE
Upgrade all packages

### PR DESCRIPTION
**Pull request by Oversan**
All packages are updated for current versions.
I have checked production and development builds.

Webpack 4 requires installation webpack-cli and modification config for mode description.

### CODE REVIEW

**Comment by AlexanderPrendota** [comment]
Hey, @Oversan. Thanks for contributing! I need some of the time to check your changes on my side! 🙂

**Comment by Oversan** [comment]
Ok. Let me know if something needs to be added or changed.

File: package.json [hunk 1]
```diff
@@ -14,45 +14,46 @@
...
   ],
   "devDependencies": {
-    "@babel/cli": "^7.1.2",
-    "@babel/core": "~7.3.0",
-    "@babel/plugin-transform-runtime": "~7.3.0",
-    "@babel/preset-env": "~7.3.0",
-    "@babel/runtime-corejs2": "~7.3.0",
-    "babel-loader": "v8.0.0-beta.0",
+    "@babel/cli": "^7.4.4",
+    "@babel/core": "~7.4.4",
```
Comment (AlexanderPrendota) [inline package.json]: @fgutmann Hey! Sorry for disturbing. As I remember we have a few problems with upgrading babel/core after version 7.4. Do you remember what kind of problems? Thanks. 🙂 It was some of your PR.

Comment (fgutmann) [reply package.json]: Hi! The issue was that babel-preset-env had a transitive dependency to core-js 3 introduced. This has an incompatible directory structure with core-js 2 and broke our build. Babel 7.4 is officially compatible with core js 2 though. A workaround back then was to add an explicit dependency to core-js 2.

I guess that the dependency from babel-preset-env to core-js 3 was a bug on the babel side. The dependencies of that package seem to have changed in the meantime. Back then it was very early days for babel 7.4. So all of that might not be an issue anymore.

See here for reference of the old issue: https://github.com/JetBrains/kotlin-playground/pull/48#discussion_r268667184

Comment (AlexanderPrendota) [reply package.json]: Thanks a lot 🙂

**Comment by Oversan** [comment]
Is there other problems with versions upgrade? After this PR merge I can send PR with test runner.

**Comment by AlexanderPrendota** [comment]
Hello @Oversan! Did u run the component via `yarn start`? 
I got a lot of errors in console when I press a run-button 😔
Screen Shot 2019-05-27 at 11 02 55 AM

**Comment by Oversan** [comment]
Yes, I had been checked application with `yarn start` and it is strange as for me look current errors.
I found what is problem. New version of package 'deepmerge' breaks flow.
I made rollback version of this version.
@AlexanderPrendota please check again.

**Comment by AlexanderPrendota** [comment]
Awesome! Let me have a look =)

**Comment by AlexanderPrendota** [comment]
@kisenka Could you please take a look on changes?

**Comment by AlexanderPrendota** [comment]
@kisenka Please don't merge that PR after approval I need to release a patch version before that

**Comment by Oversan** [comment]
@AlexanderPrendota , @kisenka may be I can help in some way?

**Comment by AlexanderPrendota** [comment]
No thanks) We are waiting for a kisenka review =)

**Comment by AlexanderPrendota** [comment]
Hey! I merged as [`2e671c7`](https://github.com/JetBrains/kotlin-playground/commit/2e671c7fe310c534afba367d1a0fa63343560828) 'cause your branch has a conflict in yarn.lock file. 

thanks for your help 👍

**Pull request closed without merging**